dali/python/nvidia/dali/_utils/autoserialize.py

```diff
@@ -57,14 +57,14 @@ def invoke_autoserialize(head_module, filename):
 
     :param head_module: Module, denoting the model tree in which the decorated function shall exist.
     :param filename: Path to the file, where the output of serialization will be saved.
-    """
+    """  # noqa W505
```

Comment:
Is the `flake8` prompting `W505` here? I've ran it on my side and didn't see such check being false. These are what I've got on `autoserialize.py` file:
```
dali/python/nvidia/dali/_utils/autoserialize.py:49:101: E501 line too long (106 > 100 characters)
dali/python/nvidia/dali/_utils/autoserialize.py:52:101: E501 line too long (102 > 100 characters)
dali/python/nvidia/dali/_utils/autoserialize.py:53:101: E501 line too long (101 > 100 characters)
dali/python/nvidia/dali/_utils/autoserialize.py:68:13: F541 f-string is missing placeholder
```

Reply:
Guess it did. I don't have `E501` or `F541` there. `flake8 --version
4.0.1 (mccabe: 0.6.1, pycodestyle: 2.8.0, pyflakes: 2.4.0) CPython 3.8.10 on Linux`